src/components/Toolbelt/Toolbelt.js

```diff
@@ -6,79 +6,61 @@ import classNames from 'classnames'
 import './Toolbelt.sass'
-import { fieldMode } from '../../enums'
-import { tools, pixel } from '../../img'
+import { tools as toolImages, pixel } from '../../img'
+import toolsData from '../../data/tools'
 
-const { CLEANUP, HARVEST, WATER } = fieldMode
+const tools = Object.values(toolsData).sort(t => t.order)
 
 const noop = () => {}
 
 export const Toolbelt = ({
   fieldMode: currentFieldMode,
   handleFieldModeSelect,
-}) => (
-  <div className="Toolbelt">
-    <div className="button-array">
-      {[
-        {
-          alt: 'A watering can for hydrating plants.',
-          fieldMode: WATER,
-          toolImageId: 'watering-can',
-          hiddenText: 'Select the watering can to water your crops',
-        },
-        {
-          alt: 'A scythe for crop harvesting.',
-          fieldMode: HARVEST,
-          toolImageId: 'scythe',
-          hiddenText: 'Select the scythe to harvest ripened crops',
-        },
-        {
-          alt:
-            'A hoe for removing crops and disposing of them. Also returns replantable items to your inventory.',
-          fieldMode: CLEANUP,
-          toolImageId: 'hoe',
-          hiddenText:
-            'Select the hoe to clear crops, or replantable field items',
-        },
-      ].map(({ alt, fieldMode, hiddenText, toolImageId }, i) => (
-        <Tooltip
-          {...{
-            key: fieldMode,
-            placement: 'top',
-            title: (
-              <>
-                <p>{alt}</p>
-                <p>(shift + {i + 1})</p>
-              </>
-            ),
-          }}
-        >
-          <Button
+}) => {
+  return (
+    <div className="Toolbelt">
+      <div className="button-array">
+        {tools.map(({ alt, fieldMode, fieldKey, hiddenText, id }) => (
+          <Tooltip
             {...{
-              className: classNames({
-                selected: fieldMode === currentFieldMode,
-              }),
-              color: 'primary',
-              onClick: () => handleFieldModeSelect(fieldMode),
-              variant: fieldMode === currentFieldMode ? 'contained' : 'text',
+              key: fieldMode,
+              placement: 'top',
+              title: (
+                <>
+                  <p>{alt}</p>
+                  <p>({fieldKey})</p>
+                </>
+              ),
             }}
           >
-            {/* alt is in a different format here because of linter weirdness. */}
-            <img
+            <Button
               {...{
-                className: `square ${toolImageId}`,
-                src: pixel,
-                style: { backgroundImage: `url(${tools[toolImageId]}` },
+                className: classNames({
+                  selected: fieldMode === currentFieldMode,
+                }),
+                color: 'primary',
+                onClick: () => handleFieldModeSelect(fieldMode),
+                variant: fieldMode === currentFieldMode ? 'contained' : 'text',
+                'data-tool-id': id,
```

Comment:
What's this used for? I don't see it referenced anywhere else. Should it be removed?

Reply:
ahh, yes, i thought i was going to use this for the unit tests at one point before we switched to react testing lib